<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBEffectPage</class>
 <widget class="QWidget" name="OpenRGBEffectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="EffectDetails">
    <item>
     <widget class="QLabel" name="EffectName">
      <property name="text">
       <string>EffectName</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="EffectDesciption">
      <property name="text">
       <string>EffectDescription</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>611</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="EffectOptions"/>
  </widget>
  <widget class="QTableWidget" name="SelectDevices">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>291</width>
     <height>431</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
